gramps/plugins/view/geomoves.py

```diff
@@ -659,28 +660,26 @@ def animate_moves(self, index, person, color):
                     death = high_date
             new_list.append([level, plxp, birth, death])
         pidx = 0
-        if isinstance(color, str):
-            color = Gdk.color_parse(color)
```

Comment:
This function (`animate_moves`) is only a callback from a timeout added in `_createmap`, or a timeout added by itself, so we know the input is a `Gdk.RGBA`, and this check can be removed.